deployments/charts/kuma/values.yaml

```diff
@@ -225,6 +225,34 @@ ingress:
   # -- Affinity placement rule for the Kuma Ingress pods
   affinity: {}
 
+egress:
+  # -- If true, it deploys Egress for cross cluster communication
+  enabled: false
+  # -- Time for which old listener will still be active as draining
+  drainTime: 30s
+  # -- Number of replicas of the Egress
+  replicas: 1
+  service:
+    # -- Service type of the Egress
+    type: LoadBalancer
```

Comment:
LoadBalancer? I believe this should be ClusterIP

Reply:
addressed